src/code/ConvertBase64Command.cs

```diff
@@ -26,6 +26,10 @@ public sealed class ConvertFromBase64Command : PSCmdlet
 
         protected override void ProcessRecord()
         {
+            while (EncodedText.Length % 4 != 0)
+            {
+                EncodedText += "=";
```

Comment:
this could be pretty ruinous from a resource/performance perspective, it could result in creating 4 new strings. If those strings are large - that could be pretty impactful.